autoagora_agents/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022-, Semiotic AI, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any
+
+import sacred
+
+from autoagora_agents.controller import Controller
+
+algorithm_ingredient = sacred.Ingredient("algorithm")
+
+
+@algorithm_ingredient.capture
+def controller(*, agents: list[dict[str, Any]], seed: int, **kwargs) -> Controller:
+    """Construct a controller from the algorithm config.
```

Comment:
not sure why we need `kwargs` in here...?

Reply:
Good call. Latest commit fixes this.